js/views/homeVw.js

```diff
@@ -422,6 +435,7 @@ module.exports = baseVw.extend({
       this.searchItems(targetText);
       addressText = addressText ? "#" + addressText.replace(/\s+/g, '') : "";
       target.val(addressText);
+      this.$el.find('.js-homeListingToggle').addClass('hide');
```

Comment:
You don't need this code, you can't type in a field unless it had focus, so the on focus code above is enough.